functions/snippets/pom.xml

```diff
@@ -26,20 +26,32 @@
   </properties>
 
   <dependencies>
-    <!-- Required to for com.example.functions.ParseContentType -->
+    <!-- Required for com.example.functions.ParseContentType -->
     <dependency>
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
       <version>2.8.6</version>
     </dependency>
 
-    <!-- Required to for com.example.functions.RetrieveLogs -->
+    <!-- Required for com.example.functions.RetrieveLogs -->
```

Comment:
I would hope this isn't true.  Can't you also write to stderr / stdout?

Reply:
That particular sample shows how to _retrieve_ logs from to Stackdriver.

The existing samples use `java.util.logging.Logger` though - would you want me to replace that with `System.out.println`?

Reply:
No - My comment wasn't to discourage use, only that most users can just use stdout/stderr and get it logged.  It's only if you want something specialized do you need Stackdriver.  

Perhaps a different comment would be in order, but that's up to you.

Reply:
Ah - so you're thinking of the `functions_log_helloworld` sample in `LogHelloWorld.java`.

Fair point - I'll add some non-Stackdriver examples there (in a separate PR).